src/gcodeExport.h

```diff
@@ -96,6 +96,7 @@ class GCodeExport : public NoCopy
     Point3 currentPosition;
     double currentSpeed; //!< The current speed (F values / 60) in mm/s
     double current_acceleration; //!< The current acceleration in the XY direction (in mm/s^2)
+    double current_travel_acceleration; //!< The current acceleration in the XY direction used for travel moves if different from current_acceleration (in mm/s^2)
```

Comment:
Would be nice to mention that this is only used by Repetier firmware at the moment.

Not blocking. This PR seems ready to be merged.